yt/frontends/arepo/fields.py

```diff
@@ -134,3 +148,13 @@ def _cr_energy_density(field, data):
                 sampling_type="local",
                 units=self.ds.unit_system["pressure"],
             )
+
+            def _cr_pressure(field, data):
+                return data["gas", "cr_energy_density"] / 3.0
```

Comment:
Why `/ 3.0`?

Reply:
The cosmic rays (CRs) are an ideal relativistic gas with gamma = 4/3, so P = (gamma-1)*internal_energy_density = 1/3*internal_energy_density.

However, I just remembered that this is a parameter that you can set in the parameter file (even though it is nearly always 1/3), so I should probably make this more general.

Reply:
Which means it probably needs a test dataset